Don't require pkg-config gem at runtime, as it is only used for building.

Index: .metadata
--- .metadata.orig
+++ .metadata
@@ -34,7 +34,7 @@ dependencies:
     - - "~>"
       - !ruby/object:Gem::Version
         version: '1.4'
-  type: :runtime
+  type: :development
   prerelease: false
   version_requirements: !ruby/object:Gem::Requirement
     requirements:
